src/mono/sample/Directory.Build.props

```diff
@@ -6,13 +6,17 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(BrowserHost)' != 'windows'">
-    <ExecXHarnessCmd>$HARNESS_RUNNER</ExecXHarnessCmd>
-    <XHarnessOutput>$XHARNESS_OUT</XHarnessOutput>
+    <ExecXHarnessVar>$HARNESS_RUNNER</ExecXHarnessVar>
+    <XHarnessOutputVar>$XHARNESS_OUT</XHarnessOutputVar>
+    <XHarnessCommandVar>$XHARNESS_COMMAND</XHarnessCommandVar>
+    <WasmXHarnessArgsVar>$XHARNESS_ARGS</WasmXHarnessArgsVar>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(BrowserHost)' == 'windows'">
-    <ExecXHarnessCmd>%HARNESS_RUNNER%</ExecXHarnessCmd>
-    <XHarnessOutput>%XHARNESS_OUT%</XHarnessOutput>
+    <ExecXHarnessVar>%HARNESS_RUNNER%</ExecXHarnessVar>
```

Comment:
Why the rename? The value is not required to be a variable, isn't it?

Reply:
this is just a way how to differentiate windows/linux way how variables are expanded when you are writing a script. It is NOT the actual value. That value will get expanded later on helix. 
That's why the rename, it was confusing me.
I will add comment to that.